### ISSUE TITLE
GreenteaOS vs ReactOS - similar goal, different ways?

### ISSUE THREAD

**Issue by informer2016** [GreenteaOS/Greentea#5]
Hi there @PeyTy , I'm curious about the feature comparison between GreenteaOS and ReactOS. Please tell, what GreenteaOS can do that ReactOS couldn't?

**Comment by ROCKNROLLKID** [comment]
From what I can see as of right now, the differences seem to be this:

What Greentea has:
- SMP support (running on multiple cores)
- Multi-threading
- Partial UEFI support (ReactOS has none)

What ReactOS has:
- USB 1.1 and 2.0 support
- A very extensive freeloader able to boot multiple file systems, xbox, and now pc-98 systems
- WinBTRFS (mainly the ability to use more then 4 gb file sizes, greentea seems to just use FastFat)
- AHCI/SATA support
- Direct X 9 and OpenGL support (greentea seems to only have software rendering)
- A large community with more developers and one that is currently being paid to fix usb/pnp bugs (greentea only has 1 developer right now)

The bottom line is that as of the time of this post, greentea beats reactos in performance, but reactos can run on much more hardware and can run much more software.

**Comment by MarcoRavich** [comment]
I still believe that a "more winning" approach would be what I suggested years ago @ [MSFN's Windows 9x Member Projects](https://msfn.org/board/forum/91-windows-9x-member-projects/): gradual substitution of - adbandoned, such 9x, Vista and now 7 too - working Windows installs...

...it's basically the same approach that Stallman had with Unix !
